README.md

```diff
@@ -10,10 +10,18 @@ Until a decent manual is written, one can always look in the `template.tex` file
 
 If using [`academicons`](https://ctan.org/tex-archive/fonts/academicons) in the template, you need to use a Xe(La)TeX or Lua(La)TeX engine for them to render. Otherwise an alternative will be used automatically.
 
+## Development
+
+[![Build template](https://github.com/moderncv/moderncv/actions/workflows/build-pdf.yml/badge.svg)](https://github.com/moderncv/moderncv/actions/workflows/build-pdf.yml)
+[![CTAN](https://img.shields.io/ctan/v/moderncv.svg)](https://www.ctan.org/pkg/moderncv)
```

Comment:
Yeah I know, but honestly I don't see why it would make sense in this case - usually the README is for devs, but here it is also for users (linked on CTAN e.g.), and there I find it more distracting.